doc/docs.md

```diff
@@ -444,25 +444,40 @@ a, b = b, a
 println('${a}, ${b}') // 1, 0
 ```
 
-### Declaration errors
+### Warnings and declaration errors
```

Comment:
Need to change line 69 as well, so the TOC is correct.

Reply:
done.Thanks for pointing it out.